package.json

```diff
@@ -12,10 +12,11 @@
     "docusaurus": "docusaurus",
     "lint": "eslint .",
     "lint-prose": "cp .proselintrc ~/ && proselint docs blog",
+    "prepare": "husky install",
```

Comment:
This is how [Husky 7 tells you to configure your project](https://www.npmjs.com/package/husky).